code/modules/hydroponics/seeds.dm

```diff
@@ -115,10 +125,10 @@
 	copy_seed.desc = desc
 	copy_seed.productdesc = productdesc
 	copy_seed.genes = list()
-	for(var/datum/plant_gene/plant_genes in genes)
-		copy_seed.genes += plant_genes.Copy()
-	for(var/datum/plant_gene/trait/traits in genes)
-		traits.on_new_seed(copy_seed)
+	for(var/datum/plant_gene/gene as anything in genes)
```

Comment:
in an earlier comment, you say that list/genes could have typepaths in it. Is this not a problem?

Reply:
I was actually double checking these last night if they'd be a problem, they PROBABLY wont, but I may remove them anyways just in case